src/agent/debugger/src/debug_event.rs

```diff
@@ -41,62 +41,66 @@ impl<'a> Display for DebugEventInfo<'a> {
             Exception(info) => {
                 write!(
                     formatter,
-                    "event=Exception exception_code=0x{:08x} exception_address=0x{:08x} first_chance={}",
+                    "Exception code=0x{:08x} address=0x{:08x} {}",
                     info.ExceptionRecord.ExceptionCode,
                     info.ExceptionRecord.ExceptionAddress as u64,
-                    info.dwFirstChance != 0
+                    if info.dwFirstChance == 0 {
+                        "second_chance"
+                    } else {
+                        "first_chance"
+                    }
                 )?;
             }
-            CreateThread(_info) => {
-                write!(formatter, "event=CreateThread")?;
+            CreateThread(info) => {
+                write!(formatter, "CreateThread handle={:x}", info.hThread as u64)?;
```

Comment:
Probably want `handle=0x{:x}` to avoid confusion.

Reply:
It's actually more useful without the `0x` when searching by word, e.g. in vim, typing `*`, it will match the `process.thread` text used in other messages.